README.md

```diff
@@ -41,27 +41,27 @@ To combine DPoS and PoA for consensus, Binance Smart Chain implement a novel con
  
 ### Light Client of Binance Chain  
```

Comment:
Can we also rename `Binance Chain` to BNB Beacon Chain?

Reply:
accepted